lib/seattleflu/id3c/cli/command/etl/redcap_det_swab_n_send.py

```diff
@@ -0,0 +1,575 @@
+"""
+Process REDCap DET documents into the relational warehouse.
+
+Contains some hard-coded logic for which project ID correlates to which project
+(e.g. 17421 is the PID for Shelters)
+"""
+import os
+import re
+import click
+import json
+import hashlib
+import requests
+from typing import Any, List, Dict, Optional, Match, Collection
+from datetime import datetime, timezone
+from id3c.db.session import DatabaseSession
+from id3c.db.datatypes import Json
+from id3c.cli.command.etl import etl, race
+
+from id3c.cli.command.etl.redcap_det import (
+    COMPLETE_AND_VERIFIED,
+    LOG,
+    HEADERS,
+    REVISION,
+    mark_skipped,
+    mark_processed,
+    get_redcap_record
+)
+
+REDCAP_URL = 'redcap.iths.org'
+INTERNAL_SYSTEM = "https://seattleflu.org"
+UW_CENSUS_TRACT = '53033005302'
+
+@etl.command("redcap-det-swab-n-send", help = __doc__)
```

Comment:
Since we have the `redcap-det` group setup in https://github.com/seattleflu/id3c/pull/73, this can be changed to `@redcap-det.command('swab-n-send')`

Reply:
Yep, that was the plan for it and no reason I can't make it a subcommand now.